currentRangesTest.py

```diff
@@ -0,0 +1,21 @@
+import unittest
+import currentRanges
+
+class CurrentRangesTest(unittest.TestCase):
+  def test_count_min(self):
+    self.assertTrue(currentRanges.count_min([4,4,5,5,4,8]) == [4,3])
+  def test_remove_min(self):
+    self.assertTrue(currentRanges.remove_min([4,4,5,5,4,8]) == [5,5,8])
+  def test_def_range_count(self):
+    self.assertTrue(currentRanges.def_range_count([4,4,5,5,4,8]) == [5,{4:5}])
```

Comment:
Suppose you get a requirement to include the single value 8 with a count of `1`.
Try writing a test for that and modify the implementation.